Modules/Relationships/src/Relationships.Infrastructure/Persistence/Database/Repository/RelationshipsRepository.cs

```diff
@@ -67,42 +60,30 @@ public class RelationshipsRepository : IRelationshipsRepository
 
         var changes = await query.OrderAndPaginate(d => d.CreatedAt, paginationFilter, cancellationToken);
 
-        await FillContentOfChanges(changes.ItemsOnPage);
-
         return changes;
     }
 
     public async Task<Relationship> FindRelationship(RelationshipId id, IdentityAddress identityAddress,
-        CancellationToken cancellationToken, bool track = false, bool fillContent = true)
+        CancellationToken cancellationToken, bool track = false, bool fillContent = true) // todo: fill content is redundant
```

Comment:
No todo comments

Reply:
Comment removed, but should we remove the input parameter?

Reply:
Sure.

Reply:
Oh, wrong place

Reply:
Ähm, not the wrong place. :) So yeah, you should still remove the parameter.